cmake/arbor-config.cmake.in

```diff
@@ -0,0 +1,9 @@
+include(CMakeFindDependencyMacro)
```

Comment:
This file used when building Arbor to make the cmake configuration that will be installed then consumed by the CMake for projects that `find_package(arbor)`?
The result will be written to `<install>/cmake/arbor/arbor-config.cmake`?

Reply:
That's correct (well, path should be `<install>/lib/cmake/arbor/arbor-config.cmake`).